templates/grafana_agent.yaml

```
@@ -0,0 +1,102 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: grafana-agent
+  namespace: kube-system
```

its likely wise to label each object in here with something so you know you can remove it when the charm is torn out

you'd like something on each of these K8s Resources so you can remove it later without having to load your template. 

```suggestion
metadata:
  name: grafana-agent
  namespace: kube-system
  labels:
  - juju.io/application: {app_name}
```

this lets you delete later with a simple "delete anything with the label `juju.io/application: kube-ovn`